doc/api/perf_hooks.md

```diff
@@ -219,6 +240,27 @@ The high resolution millisecond timestamp at which the Node.js process
 completed bootstrapping. If bootstrapping has not yet finished, the property
 has the value of -1.
 
+### performanceNodeTiming.eventLoopDelay
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The accumulated event loop delay. This value is only updated after the
```

Comment:
The docs don't describe what event loop delay is, and they should. I'm reading the code to see what it is.